sdk/component/reporter_test.go

```diff
@@ -127,7 +127,7 @@ func TestRunReporter(t *testing.T) {
 				Return(nil),
 		)
 
-		err := runReporterHelper(t, ctx, workflowID, mockReporter, mockStore)
+		err := runReporterHelper(t, ctx, instanceID, mockReporter, mockStore)
 		require.Error(t, err)
```

Comment:
all these instances should be `require.ErrorIs` instead of splitting it into 2 checks